yorkie/src/androidTest/kotlin/dev/yorkie/core/DocumentTest.kt

```diff
@@ -621,4 +623,215 @@ class DocumentTest {
             collectJobs.values.forEach(Job::cancel)
         }
     }
+
+    @Test
+    fun test_single_document_event_stream_can_emit_document_status_changed() {
```

Comment:
We doesn't implement this behavior before, but I added this bacause it's needed, I think.